app/src/main/java/org/android/go/sopt/week1/presentation/MainActivity.kt

```diff
@@ -0,0 +1,43 @@
+package org.android.go.sopt.week1.presentation
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import org.android.go.sopt.R
+import org.android.go.sopt.databinding.ActivityMainBinding
+import org.android.go.sopt.week1.presentation.gallery.GalleryFragment
+import org.android.go.sopt.week1.presentation.home.HomeFragment
+import org.android.go.sopt.week1.presentation.search.SearchFragment
+import org.android.go.sopt.week1.utill.binding.BindingActivity
+
+class MainActivity : BindingActivity<ActivityMainBinding>(R.layout.activity_main) {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        initTransactionEvent()
+    }
+
+    private fun initTransactionEvent() {
+        binding.bnvMain.setOnItemSelectedListener {
+            when (it.itemId) {
+                R.id.menu_home -> {
+                    changeFragment(HomeFragment.newInstance())
+                }
+                R.id.menu_gallery -> {
+                    changeFragment(GalleryFragment.newInstance())
+                }
+                R.id.menu_search -> {
+                    changeFragment(SearchFragment.newInstance())
+                }
```

Comment:
프래그먼트를 이렇게 전달하는 이유는 BindingFragment를 사용하기 때문인가요?
``
```suggestion
    private val HomeFragment by lazy { HomeFragment() }
    private val GalleryFragment by lazy { GalleryFragment() }
    private val SearchFragment by lazy { SearchFragment() }
                R.id.menu_home -> {
                    changeFragment(HomeFragment)
                }
                R.id.menu_gallery -> {
                    changeFragment(GalleryFragment)
                }
                R.id.menu_search -> {
                    changeFragment(SearchFragment)
                }
```
이렇게 by 속성 위임을 사용하면 그대로 사용할 수 있을 것 같습니다..!!